src/compressed_tensors/quantization/lifecycle/forward.py

```diff
@@ -304,6 +327,8 @@ def maybe_calibrate_or_quantize(
         # skip quantization
         return value
 
+    g_idx = getattr(module, "weight_g_idx", None)
```

Comment:
nit
id put this right before you call fake_q. You dont use it till then

Reply:
`g_idx` is an argument to the the `observer.forward` calls, but that's implemented in the follow up PR #97

Reply:
ok i see g_idx is used in between